### ISSUE TITLE
WSL2 docker import of xz files

### ISSUE THREAD

**Issue by ryanmerolle** [docker/for-win#10364]
- [X] I have tried with the latest version of Docker Desktop
  - [X] I have tried disabling enabled experimental features
  - [X] I have uploaded Diagnostics
  - Diagnostics ID: 842BFD4D-EDFA-4157-AAD7-1A787D4805F0/20210211191014

### Actual behavior

When importing a tar.xz using docker import, in a command prompt, WSL (Ubuntu 20.04), or powershell, I get the same error message that xz is not in the path even though it is.  When I manually decompress the file using xz, it works, and I can import the image.

```
$ docker import  cEOS-lab-4.24.4M.tar.xz ceos:4.24.4M
Error response from daemon: exec: "xz": executable file not found in $PATH

$ echo $PATH
[omitted]:/usr/sbin:/usr/bin:[omitted]

$ which xz
/usr/bin/xz

$ xz --version
xz (XZ Utils) 5.2.4
liblzma 5.2.4

$ xz -d cEOS-lab-4.24.4M.tar.xz

$ docker import cEOS-lab-4.24.4M.tar ceos:4.24.4M
sha256:69fa3f7649c498a00ab5d952e2d91ca9744ca585bf4ef03a711a85539e5d5b37

$ docker image ls
REPOSITORY          TAG                 IMAGE ID            CREATED              SIZE
ceos                4.24.4M             69fa3f7649c4        About a minute ago   1.68GB

```

### Expected behavior

When importing docker images compressed with xz, it should just work like the below.  It worked before I moved to WSL2 about a year ago.

```
$ docker import cEOS-lab-4.24.4M.tar.xz ceos:4.24.4M
sha256:69fa3f7649c498a00ab5d952e2d91ca9744ca585bf4ef03a711a85539e5d5b37

$ docker image ls
REPOSITORY          TAG                 IMAGE ID            CREATED              SIZE
ceos                4.24.4M             69fa3f7649c4        About a minute ago   1.68GB
```

### Information
WSL2 with docker desktop 3.1.0 & no experimental features enabled.

  - Windows Version: Version	10.0.19042 Build 19042
  - Docker Desktop Version :3.1.0
  - Backend: WSL2
  - Desktop Type: Physical Windows Desktop (not a VM)

### Steps to reproduce the behavior
I cannot provide the container image that I tested with because of some licensing associated, but I was able to reproduce this issue on other systems setup similarly.  From the looks of the error, you should be able to reproduce with any xz  compressed image.

  1. docker import image.tar.xz

**Comment by stephen-turner** [comment]
Thanks for the report. This is the same as docker/for-mac#5271. A fix is in progress.

**Comment by stephen-turner** [comment]
This is fixed in versions 3.2.0 and 3.2.1. Closing the ticket now.
